website/cue/reference/components/sinks/azure_blob.cue

```diff
@@ -68,12 +68,23 @@ components: sinks: azure_blob: {
 
 	configuration: {
 		connection_string: {
-			description: "The Azure Blob Storage Account connection string. Only authentication with access key supported."
-			required:    true
+			description: "The Azure Blob Storage Account connection string. Only authentication with access key supported. This or storage_account has to be provided."
+			required:    false
+			common:      true
 			type: string: {
+				default: ""
 				examples: ["DefaultEndpointsProtocol=https;AccountName=mylogstorage;AccountKey=storageaccountkeybase64encoded;EndpointSuffix=core.windows.net"]
 			}
 		}
+		storage_account: {
```

Comment:
I'm not crazy about the name here, it seems like this option would eventually be used for other `azure_*` components for authentication, right?

Would something like `identity` or `credential` 🤔

Reply:
That's the name of the storage account to connect to and isn't use to connect to e.g. LogAnalytics. The storage account is where stuff is stored.
The identity used to connect to it isn't actually in the config. With the env vars you can pass user and password/certificat. With cli or managed identity the client just gets a bearer token which it uses to connect.